playbook/app/pb_kits/playbook/pb_collapsible/collapsible_main.html.erb

```diff
@@ -5,15 +5,19 @@
         <%= content.presence %>
       <% end %>
       <%= pb_rails("flex/flex_item") do %>
-        <div style="color: <%= object.icon_color %>" class="icon_wrapper" >
-        <% if object.icon.present? %>
-          <%= pb_rails("icon", props: { icon: object.show_icon(object.icon)[0], id:"collapsible_open_icon", size: object.size }) %>
-          <%= pb_rails("icon", props: { icon: object.show_icon(object.icon)[1], id:"collapsible_close_icon", size: object.size }) %>
+        <% if object.icon == "none" %>
+          <div></div>
```

Comment:
Hey, @ElisaShapiro, to make sure: is this empty div intentional?

Reply:
Good catch and good question - I initially intended it as a way to preserve the space leftover after removing the icon/icon-wrapper (which needed explicit removal, or icon-related pseudo states appeared on top of the iconless area). There's a similar set up in CollapsibleMain.tsx as well. @nidaqg and I synced over zoom - we discussed taking the empty step to remove the empty flex item entirely when no icon but also considered that there might be a hidden benefit to preserving the two flex item structure. I have no preference.